netdata/netdata
# Add a chart for out of memory kills

##### Summary
SSIA

Closes #10875

##### Component Name
proc plugin

##### Test Plan
1. Check the `mem.oom_kill` chart.
2. Run a container with memory limit `docker run -m 6m mongo`.
3. Check if the OOM kill was displayed in the chart.
